app/controllers/AdminUsers/orders_controller.rb

```diff
@@ -6,9 +6,11 @@ def index
 
     def update
       if order.status_not_delivered?
-        order.status_delivered!
-        redirect_back fallback_location: admin_users_employees_path,
-                      notice: 'The order has been delivered successfully!'
+        if order.status_delivered!
```

Comment:
I find this action a bit weird - you are running an update action without any parameters which is changing the status and sending an email.
I would pass a status parameter, or create a custom action

Reply:
Understood! I have added status parameter into method ;)